base/commands/serializer/validation_schema.go

```diff
@@ -0,0 +1,65 @@
+package serializer
+
+const validationSchema = `{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "https://github.com/hazelcast/hazelcast-client-protocol/blob/master/schema/protocol-schema.json",
+  "title": "Hazelcast Client Protocol Definition",
```

Comment:
```suggestion
  "title": "Hazelcast Compact Serialization Schema",
```

Reply:
Shall we remove `validationSchema` as it is not used anymore, or keep it for future?